pilot/pkg/serviceregistry/kube/register_test.go

```diff
@@ -112,3 +112,108 @@ func TestAddLabelsAndAnnotations(t *testing.T) {
 		t.Errorf("Got unexpected %v for annotation a1=av1", o.Annotations["a1"])
 	}
 }
+
+//func TestRegisterEndpoint(t *testing.T) {
```

Comment:
We can go ahead and merge this if you remove commented out test and add in follow on PR (or we can wait with this PR till these tests can be un-commented).